code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -260,6 +281,7 @@ var/list/robot_verbs_default = list(
 			module_sprites["Basic"] = "Engineering"
 			module_sprites["Antique"] = "engineerrobot"
 			module_sprites["Landmate"] = "landmate"
+			module_sprites["Hoverdrone"] = "drone-engineer"

```

Comment:
Wait-wait-wait-wait; Are these existing sprites? Small floating ball sprites? Because those are fluff sprites for one specific player, not intended to be given to anyone but said player.